sdk/storage/Azure.Storage.Shared/tests/TestConfigurationsTemplate.xml

```diff
@@ -0,0 +1,64 @@
+<TestConfigurations>
+  <TargetTestTenant>ProductionTenant</TargetTestTenant>
+  <TargetPremiumBlobTenant>ProductionTenant</TargetPremiumBlobTenant>
+  <TargetSecondaryTestTenant>ProductionTenant2</TargetSecondaryTestTenant>
+  <TargetPreviewBlobTenant>NotInPreview</TargetPreviewBlobTenant>
+  <TargetOAuthTenant>OAuthTenant</TargetOAuthTenant>
+  <TenantConfigurations>
+    <TenantConfiguration>
+      <TenantName>DevStore</TenantName>
+      <TenantType>DevStore</TenantType>
+      <AccountName>devstoreaccount1</AccountName>
+      <AccountKey>Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==</AccountKey>
+      <BlobServiceEndpoint>http://127.0.0.1:10000/devstoreaccount1</BlobServiceEndpoint>
+      <QueueServiceEndpoint>http://127.0.0.1:10001/devstoreaccount1</QueueServiceEndpoint>
+      <TableServiceEndpoint>http://127.0.0.1:10002/devstoreaccount1</TableServiceEndpoint>
+    </TenantConfiguration>
+    <TenantConfiguration>
+      <TenantName>ProductionTenant</TenantName>
+      <TenantType>Cloud</TenantType>
+      <AccountName>[ACCOUNT NAME HERE]</AccountName>
+      <AccountKey>[ACCOUNT KEY HERE]</AccountKey>
+      <BlobServiceEndpoint>http://[ACCOUNT].blob.core.windows.net</BlobServiceEndpoint>
+      <QueueServiceEndpoint>http://[ACCOUNT].queue.core.windows.net</QueueServiceEndpoint>
+      <TableServiceEndpoint>http://[ACCOUNT].table.core.windows.net</TableServiceEndpoint>
+      <FileServiceEndpoint>http://[ACCOUNT].file.core.windows.net</FileServiceEndpoint>
+      <BlobServiceSecondaryEndpoint>http://[ACCOUNT]-secondary.blob.core.windows.net</BlobServiceSecondaryEndpoint>
+      <QueueServiceSecondaryEndpoint>http://[ACCOUNT]-secondary.queue.core.windows.net</QueueServiceSecondaryEndpoint>
+      <TableServiceSecondaryEndpoint>http://[ACCOUNT]-secondary.table.core.windows.net</TableServiceSecondaryEndpoint>
```

Comment:
Add one for FileServiceSecondaryEndpoint

Reply:
and elsewhere

Reply:
Fixed